src/dmd/globals.d

```diff
@@ -76,6 +76,13 @@ enum CPU
     native              // the machine the compiler is being run on
 }
 
+enum PIC : ubyte
```

Comment:
Is there a better name for this? Because if it’s fixed it isn’t really PIC.

Reply:
It's levels of position independent code.